docs/lang/articles/contribution/development_tips.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 3
+---
+
+# Development tips
+
+This section gives you some tips on the Taichi compiler development.
+Please make sure you have gone through [developer installation](./dev_install).
+
+## Workflow of the Taichi compiler
+
+[Life of a Taichi kernel](./compilation.md) may be worth checking out. It
```

Comment:
"may be worth checking out" -> is a good place to get started on Taichi compiler project, which explains the whole compilation process step by step.